test/units/parsing/test_mod_args.py

```diff
@@ -126,9 +125,7 @@ def test_multiple_actions_ping_shell(self):
         with pytest.raises(AnsibleParserError) as err:
             m.parse()
 
-        assert err.value.args[0].startswith("conflicting action statements: ")
-        actions = set(re.search(r'(\w+), (\w+)', err.value.args[0]).groups())
-        assert actions == set(['ping', 'shell'])
+        assert err.value.args[0] == f'conflicting action statements: {", ".join(args_dict.keys())}'
```

Comment:
This can be simplified:

```suggestion
        assert err.value.args[0] == f'conflicting action statements: {", ".join(args_dict)}'
```